charts/quickwit/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: quickwit
 description: Open-Source & Cloud-Native Log Management at any scale.
 type: application
-version: 0.1.13
+version: 0.2.0
```

Comment:
Why bump to `0.2.0` instead of `0.1.14`? (I'm fine with both)

Reply:
it's not really a patch or bugfix, so i set as `0.2.0`. As you want :)